### ISSUE TITLE
📝  Add a release notes section to the official documentation

### ISSUE THREAD

**Issue by lishanl** [SatelCreative/spylib#158]
I think it would be nice to have a release notes section like this FastApi has [here](https://fastapi.tiangolo.com/release-notes/).  For us, we can have what's new and what's breaking for each version release.

**Comment by hillairet** [comment]
Related to #61

**Comment by hillairet** [comment]
Not clear how to automate this without making this annoying.

**Comment by hillairet** [comment]
Let's just start by manually generating the changelog using the Github release without actually creating the release.
Then once the release branch is merged we create the release and regenerate the changelog, in other words the last commit of creating the release won't be in the CHANGELOG.md but that's fine.
Edit file [Development - contributing](https://github.com/SatelCreative/spylib/blob/main/docs/development-contributing.md) accordingly.

**Comment by hillairet** [comment]
@lishanl Please find a simple way to do this as in without too many manual steps.
